scss/layout/_grid.scss

```diff
@@ -52,31 +65,29 @@
   }
 }
 
+// BREAKPOINT: MEDIUM
+@include media($med) {
```

Comment:
Kind of, the important difference is that this one applies to `$med` and up, while the previous one doesn't. Without this, the `grid--2-wide` becomes 1 column on large screens.

Not sure if there's a cleamer way to structure it, let me know if you have suggestions :)